src/python_testing/TC_RVCRUNM_2_1.py

```diff
@@ -28,6 +28,12 @@
 
 class TC_RVCRUNM_2_1(MatterBaseTest):
 
```

Comment:
Can you change the test run comment above to reference --endpoint? I can't make a suggestion on that line in this PR.